src/parsecommand/parse_command.c

```diff
@@ -4,12 +4,14 @@
 ** File description:
 ** parse char * to commands
 */
-
+#include "my_puts.h"
```

Comment:
can you put this at end ofinclude because first the <> adter the ""

Reply:
its good

Reply:
here